src/twisted/spread/pb.py

```diff
@@ -455,9 +491,11 @@ class CopiedFailure(RemoteCopy, failure.Failure):
     def printTraceback(self, file=None, elideFrameworkCode=0, detail='default'):
         if file is None:
             file = log.logfile
+        failureType = self.type
+        if not isinstance(failureType, str):
+            failureType = failureType.decode("utf-8")
```

Comment:
This is interesting.  It looks like it would only work on Python 2 unintentionally because `failureType` probably usually only contains ASCII code points?

That is, on Python 2, if failureType is not `str` then most likely it is `unicode` and `unicode_str.decode("utf-8")` is equivalent to `unicode_str.encode("ascii").decode("utf-8")` which fails if `unicode_str` contains code points which ASCII can't encode.